packages/atomic/cypress/integration/sort-dropdown.cypress.ts

```diff
@@ -0,0 +1,85 @@
+import {getApiRequestBody, getAnalytics} from '../utils/network';
+import {setUpPage} from '../utils/setupComponent';
+
+const sortDropdown = 'atomic-sort-dropdown';
+const criteria = ['relevancy', 'date descending'];
```

Comment:
I see that the requirement for criteria is `criteria: criteria expression, can be multiple criteria, separated by commas`,
so can we have a test that criteria contains more than 1 field?